File: Practice/05/C++/05/05/05.cpp
#include <iostream>
using namespace std;

int main()
{
	double x0, v0, t, g;
	g = 9.8;
	cin >> x0 >> v0 >> t;
	if (t < 0) exit(0);
	if (v0 < 0) exit(0);
	int xt = x0 + v0 * t - g * t * t / 2;
	cout << abs(xt - x0);

}

File: Practice/05/Python/05.py
x0 = int( input())
v0 = int( input())
t = int( input())
a = g = 9.8
x = x0 + v0*t - (a*t*t)/2
import sys
if v0 < 0: sys.exit(0)
if t < 0: sys.exit(0)

print(abs(x - x0))
